moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -318,6 +341,12 @@ class Robot
                                          const std::string& frame_in,
                                          const std::string& frame_out) const;
 
+  std::vector<moveit_simple::JointTrajectoryPoint> 
+    trajectoryTypeConversionFromROSToMoveItSimple(std::vector<trajectory_msgs::JointTrajectoryPoint> & ROS_joint_trajectory_points)  const; 
```

Comment:
Way too long.  The return and parameter types make this obvious.  How about `toJointTrajectoryPoint` and `toFollowJointTrajectoryGoal`.  Because parameters can be different (overrides), but return types cannot, this ensures you will be able to support future conversions.